plugins/kafka/CMakeLists.txt

```diff
@@ -18,33 +18,14 @@ VASTRegisterPlugin(
   SOURCES ${kafka_sources}
   INCLUDE_DIRECTORIES include)
 
-option(VAST_ENABLE_BUNDLED_RDKAFKA "Always use the bundled RdKafka" OFF)
-add_feature_info("VAST_ENABLE_BUNDLED_RDKAFKA" VAST_ENABLE_BUNDLED_RDKAFKA
-                 "always use the bundled RdKafka.")
-if (NOT VAST_ENABLE_BUNDLED_RDKAFKA)
-  find_package(RdKafka QUIET)
-  if (RdKafka_FOUND)
-    string(APPEND VAST_FIND_DEPENDENCY_LIST
-           "\nfind_package(RdKafka REQUIRED)")
-    dependency_summary("librdkafka" RdKafka::rdkafka++ "Dependencies")
-  endif ()
-endif ()
+find_package(RdKafka QUIET)
 if (NOT RdKafka_FOUND)
-  if (NOT VAST_ENABLE_BUNDLED_RDKAFKA)
-    message(
-      STATUS
-        "Cannot find installed RdKafka; falling back to bundled version")
-  endif ()
-  set(RDKAFKA_BUILD_EXAMPLES OFF)
-  set(RDKAFKA_BUILD_TESTS OFF)
-  if (VAST_ENABLE_STATIC_EXECUTABLE)
-    set(RDKAFKA_BUILD_STATIC ON)
-  endif ()
-  add_subdirectory(aux/librdkafka)
-  dependency_summary("librdkafka" "${CMAKE_CURRENT_SOURCE_DIR}/aux/librdkafka"
-                     "Dependencies")
-  add_library(RdKafka::rdkafka++ ALIAS rdkafka++)
+  list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+  find_package(RdKafka QUIET REQUIRED)
+  list(POP_BACK CMAKE_MODULE_PATH)
 endif ()
 
-target_link_libraries(kafka PUBLIC vast::libvast vast::internal)
+dependency_summary("librdkafka" RdKafka::rdkafka++ "Dependencies")
 target_link_libraries(kafka PUBLIC RdKafka::rdkafka++)
+target_link_libraries(kafka-shared PUBLIC RdKafka::rdkafka++)
```

Comment:
`kafka-shared` doesn't exist when `VAST_ENABLE_STATIC_PLUGINS` is true.

Reply:
Ah damn, I totally overlooked this. That's simply a leftover from debugging, we need neither this line nor the one below.